tests/suites/test_suite_ecp.function

```diff
@@ -2,6 +2,7 @@
 #include "mbedtls/ecp.h"
 #include "mbedtls/ecdsa.h"
 #include "mbedtls/ecdh.h"
+#include "ecp_invasive.h"
```

Comment:
Why do we need this? I can't see anything being added in this PR that requires this include. What am I missing?

Reply:
Accidentally removed the include of `ecp_invasive.h` when adding `ecp_internal.h` in  65fc9f78d49caa69c57c4f2074a4067937028599.